scripts/portstat

```diff
@@ -106,7 +120,7 @@ class Portstat(object):
         if speed is None:
             speed = PORT_RATE
         else:
-            speed = int(speed)//1000
+            speed = int(speed)/1000
```

Comment:
Why undo the floor division?

Reply:
didn't realize that changed.  Never meant to change anything in that.  Reverted back to normal.